ig/input/fsh/nbs-orc-segment.fsh

```diff
@@ -2,15 +2,20 @@ Logical: ORCSegment
 Id: segment-orc-logical-model
 Title: "ORC Segment"
 Description: "The ORC Segment"
-* orderControl 1..1 SU string "ORC-1 Order Control"
-* ORCPlacerOrderNumber 1..1 SU string "ORC-2 Placer Order Number"
-* fillerOrderNumber 1..1 SU string "ORC-3 Filler Order Number"
-* placerGroupNumber 1..1 SU string "ORC-4 Placer Group Number"
-* dateTimeOfTransaction 1..1 SU string "ORC-9 Date/Time of Transaction"
-* orderingProvider 1..1 SU string "ORC-12 Ordering Provider"
-* orderingFacilityName 1..1 SU string "ORC-11 prdering Facility Name"
-// other optional fields
-
+* orderControl 1..1 SU id "ORC-1 Order Control"
+* placerOrderNumber 1..1 SU Identifier "ORC-2 Placer Order Number"
```

Comment:
I wouldn't use any complex data types here because the complex types suggest additional fields and structure where there isn't in these HL7 fields.  For example, [`Identifier` states](http://hl7.org/fhir/R4/datatypes.html#identifier) it has a sub-field of `use`, `system`, and `value`.  This HL7 field doesn't have those as sub-fields.  So, I'd just stick to the [primitive types](http://hl7.org/fhir/R4/datatypes.html#primitive).

```suggestion
* placerOrderNumber 1..1 SU string "ORC-2 Placer Order Number"
```